### PR TITLE
SUPPORTENG-824 Add CTAs to ZIM page throughout platform docs

### PR DESCRIPTION
**Pull request by cameronjp**
https://zapierorg.atlassian.net/browse/SUPPORTENG-824

### CODE REVIEW

File: docs/_partners/faq.md [hunk 1]
```diff
@@ -37,6 +37,10 @@ Connecting to Zapier must be easy for the user. OAuth v2 is the preferred authen
 
 Only submit your integration for review when it is complete and ready to be published. Make sure to thoroughly test your integration for edge cases, fix all bugs, and address [integration checks](https://platform.zapier.com/docs/integration-checks-reference) before submitting. Confirm your integration adheres to our [Integration Review Guidelines](https://platform.zapier.com/partners/integration-review-guidelines) thoroughly. Please ensure both your platform and integration work consistently without errors.
 
+## User Zapier Issue Manager
+
+Use Zapier Issue Manager to sync and manage issues for your Zapier integration from your own issue-tracking tools (such as Jira or Trello). [Learn more and request access to Zapier Issue Manager here.](https://platform.zapier.com/partners/zim)
```
Comment (wavebeem) [inline docs/_partners/faq.md:42]: #nitpick I would just use "Zapier Issue Manager" as the link text, personally

**Review by wavebeem — APPROVED** [review]
